nvdaHelper/remote/winword.cpp

```diff
@@ -525,9 +526,18 @@ void generateXMLAttribsForFormatting(IDispatch* pDispatchRange, int startOffset,
 	if((formatConfig&formatConfig_reportLineNumber)&&(_com_dispatch_raw_method(pDispatchRange,wdDISPID_RANGE_INFORMATION,DISPATCH_PROPERTYGET,VT_I4,&iVal,L"\x0003",wdFirstCharacterLineNumber)==S_OK)) {
 		formatAttribsStream<<L"line-number=\""<<iVal<<L"\" ";
 	}
-	if((formatConfig&formatConfig_reportAlignment)||(formatConfig&formatConfig_reportParagraphIndentation)||(formatConfig&formatConfig_reportLineSpacing)) {
+	if((formatConfig&formatConfig_reportColor)||(formatConfig&formatConfig_reportAlignment)||(formatConfig&formatConfig_reportParagraphIndentation)||(formatConfig&formatConfig_reportLineSpacing)) {

```

Comment:
Fix formatting for readability

```suggestion
	if (
		(formatConfig & formatConfig_reportColor)
		|| (formatConfig & formatConfig_reportAlignment)
		|| (formatConfig & formatConfig_reportParagraphIndentation)
		||(formatConfig & formatConfig_reportLineSpacing)
	) {
```